docs/_plugins/pomrepo.md

```diff
@@ -72,6 +72,7 @@ The query must return a JSON response.
 | `name`           | `STRING`|       | Required name of the repo.|
 | `transitive`     | `true|false` | `true` | If set to _truthy_ then dependencies are transitive.|
 | `poll.time`      | `integer`| 5 minutes | Number of seconds between checks for changes to POM files referenced by `pom` or `revision`. If the value is negative or the workspace is in batch/CI mode, then no polling takes place.|
+| `depManagement`      | `boolean`| false | if set to true, dependencies in the `dependencyManagement` section will handlet as actul dependencies|
```

Comment:
Several misspellings.

Reply:
I hopefully fixed the it. a recheck would be appreciated.